README.md

```diff
@@ -128,6 +128,7 @@ Do the following when releasing:
    - Create a branch named `rx.y`.
    - Update the [libmongocrypt-release](https://evergreen.mongodb.com/projects##libmongocrypt-release) Evergreen project to set `Branch Name` to `rx.y`.
    - Update the Linux distribution package installation instructions in the below sections to refer to the new version x.y.
+   - In the Java binding build.gradle.kts, update the `LibMongoCryptS3Data` function. Replace the minor version `rx.y` in the URL with the new branch.
```

Comment:
This isn't going to work.  The build script has a check to ensure that the git clone is not dirty, so any manual change will cause that check to fail.  We need to add the ability to parameterize this value at build time via system property,  similar to how we do with the gitRevision.

Reply:
The tagging (git commit and git tag) is occurring after this step.